api/src/pcapi/routes/serialization/public_api_collective_offers_serialize.py

```diff
@@ -490,6 +541,19 @@ def validate_beginning_limit_datetime(cls, beginningDatetime: datetime | None) -
             raise ValueError("La date de début de l'évènement ne peut pas être nulle.")
         return beginningDatetime
 
+    def get_image_as_bytes(self) -> bytes | None:
+        if not self.imageFile:
+            return None
+        try:
+            return base64.b64decode(self.imageFile.encode("utf-8"))
+        except binascii.Error:
+            raise ApiErrors(
+                errors={
+                    "imageFile": ["La chaine ne semble pas être du base 64 valide."],
```

Comment:
```suggestion
                    "imageFile": ["La valeur ne semble pas être du base 64 valide."],
```

pour être homogène avec le message d'erreur correspondant dans `PostCollectiveOfferBodyModel`